packages/datetime/src/dateInput.tsx

```diff
@@ -182,13 +190,14 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         return (
             <Popover
                 autoFocus={false}
-                content={popoverContent}
                 enforceFocus={false}
                 inline={true}
                 isOpen={this.state.isOpen && !this.props.disabled}
-                onClose={this.handleClosePopover}
-                popoverClassName="pt-dateinput-popover"
                 position={this.props.popoverPosition}
+                {...popoverProps}
+                content={popoverContent}
+                onClose={this.handleClosePopover}
+                popoverClassName={classNames("pt-dateinput-popover", popoverProps.popoverClassName)}
```

Comment:
if i'm not mistaken, that is the purpose of this line (and why it comes after the spread)